requirements.txt

```diff
@@ -1,3 +1 @@
-expecttest
 requests
-torch
```

Comment:
I know we are still installing PyTorch nightly for testing with CI. Does the removal of `torch` from `requirements.txt` impact anything else such as installation?

Reply:
It won't because we explicitly have `torch` here https://github.com/pytorch/data/blob/012d8f279e313e4b21a387cc9bf10449ce7b3f4e/setup.py#L55